cores/arduino/HardwareTimer.cpp

```diff
@@ -883,7 +885,7 @@ void HardwareTimer::detachInterrupt()
   * @param  callback: interrupt callback
   * @retval None
```

Comment:
Can you add a comment about the new parameter (like it is done for other parameter)